### PR TITLE
Make sure break line before next command

### PR DESCRIPTION
**Pull request by leandrocrs**
Add a line break before every prompt line.

Related to [issue 21](https://github.com/pecigonzalo/Oh-My-Powershell/issues/21)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # Oh-My-Powershell
 [![Join the chat at https://gitter.im/pecigonzalo/Oh-My-Powershell](https://badges.gitter.im/pecigonzalo/Oh-My-Powershell.svg)](https://gitter.im/pecigonzalo/Oh-My-Powershell?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-After getting tired of mantaining my powershell profile, I decided to split the components and improve it, as I already use Oh-My-Zsh I thought it would be a good idea to have something similar on Powershell.
+[GitHub Page](http://leandrw.github.io/Oh-My-Powershell)
```
Comment (pecigonzalo) [inline README.md]: Maybe dont put this as part of this PR and we can put it on another one for the website.

**Comment by pecigonzalo** [comment]
Fixes #21
Fixes #19

**Comment by pecigonzalo** [comment]
@leandrw please remove or modify the line in my comment. Otherwise i cant merge this.

**Comment by leandrocrs** [comment]
@pecigonzalo done.

File: themes/blocky.ps1 [hunk 2]
```diff
@@ -2,8 +2,9 @@
     $realCommandStatus = $?
     $realLASTEXITCODE = $LASTEXITCODE
     $Path = $pwd.ProviderPath
+    Write-Host
     Write-Host " $Path " -NoNewLine -ForegroundColor Black -BackgroundColor White
-    if($gitStatus){
+    if(Get-GitStatus){
```
Comment (pecigonzalo) [inline themes/blocky.ps1]: This will cause to many unnecessary calls to the Get-Gitstatus function, $gitstatus represents if git-posh exists or not.

**Comment by pecigonzalo** [comment]
@leandrw Please read my comment about Get-GitStatus

**Comment by pecigonzalo** [comment]
+1

**Pull request merged**